hack/local-up-cluster.sh

```diff
@@ -632,6 +633,7 @@ EOF
       --etcd-servers="http://${ETCD_HOST}:${ETCD_PORT}" \
       --service-cluster-ip-range="${SERVICE_CLUSTER_IP_RANGE}" \
       --feature-gates="${FEATURE_GATES}" \
+      --emulated-version="${EMULATED_VERSION}" \
```

Comment:
nit: Can we use `--emulated-version=kube=${EMULATED_VERSION}` to be aligned with https://github.com/kubernetes/kubernetes/pull/125742? It is defaulted to kube but consistency would be nice.

Reply:
Ah good catch thanks!  I've updated the value for all three flag entries.  Now they are:
 `--emulated-version=kube=${EMULATED_VERSION}`

Reply:
I've undone this change as there was some test failures and UX issues in my testing with the "kube=" prefix added to the flag entires.  The default value for EMULATED_VERSION is "" so the value passed is `kube=""`.  Going to keep the initial syntax as adding logic to prepend a suffix or use single quotes is different than other env vars in the file and also the prepending logic might confuse users. With the initial approach users can set EMULATED_VERSION=1.31 OR EMULATED_VERSION=kube=1.31

Reply:
I've updated the script to instead use:

EMULATED_VERSION=${EMULATED_VERSION:+kube=$EMULATED_VERSION}

This way it is consistent with https://github.com/kubernetes/kubernetes/pull/125742 and has the default values we want

Reply:
perfect, thanks!

Reply:
aside: in most other scripts we make the "public API" use `KUBE_` but we should probably remain self-consistent within this script/tool